packages/contract-schema/package.json

```diff
@@ -8,20 +8,20 @@
     "build": "cd spec && json2ts -i contract-object.spec.json -o ./index.d.ts",
     "test": "mocha"
   },
-  "repository": "https://github.com/trufflesuite/truffle/tree/master/packages/truffle-contract-schema",
+  "repository": "https://github.com/trufflesuite/truffle/tree/develop/packages/contract-schema",
```

Comment:
```suggestion
  "repository": "https://github.com/trufflesuite/truffle/tree/master/packages/contract-schema",
```

Reply:
I think the `develop` branch is the default for all of the packages. Do we want them to be `master`? I guess we probably do.